docker-compose.yml

```diff
@@ -35,3 +38,15 @@ services:
       - .env
     ports:
       - 27018:27018
+  db_seed:
+    image: node:17
+    command: npx prisma db seed
+    depends_on:
+      - chronicle-database
+      - mongo_replica
+    volumes:
+      - ./server:/usr/src/app
```

Comment:
Instead of voluming everything you should just copy over the necessary files and run npx prisma db